source/features/deprioritize-marketplace-link.tsx

```diff
@@ -1,27 +1,28 @@
 import React from 'dom-chef';
 import select from 'select-dom';
 import onetime from 'onetime';
-import domLoaded from 'dom-loaded';
 import elementReady from 'element-ready';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-async function init(): Promise<void> {
-	const marketPlaceLink = (await elementReady('.Header-link[href="/marketplace"]'));
-	if (marketPlaceLink) {
-		// The Marketplace link seems to have an additional wrapper that other links don't have https://i.imgur.com/KV9rtSq.png
-		marketPlaceLink.closest('.border-top, .mr-3')!.remove();
-	}
-
-	await domLoaded;
-
+function handleMenuOpening(): void {
 	select.last('.header-nav-current-user ~ .dropdown-divider')!.before(
 		<div className="dropdown-divider"/>,
 		<a className="dropdown-item" href="/marketplace">Marketplace</a>
```

Comment:
Does GHE link here or always to github.com? If github.com, let's use this:
```suggestion
		<a className="dropdown-item" href="https://github.com/marketplace">Marketplace</a>
```

Reply:
Not sure, I've not seen it activated in the wild, just sounds possible from the release notes.

Reply:
Who knows then. The selector only specifically selects local links (`.Header-link[href="/marketplace"]`) so if this change is required, the selector also needs to be changed.

Since nobody complained, I'll leave it as is.